src/main/java/core/selenium/WebDriverConfig.java

```diff
@@ -22,12 +24,16 @@
  * @author Jesus Menacho
  * @version 1.0
  */
-public class WebDriverConfig {
+public final class WebDriverConfig {
     private String webBrowser;
     private String implicit;
     private String explicit;
```

Comment:
explicit, what? please define better the variable name

Reply:
Done.